packages/type-safe-api/docs/developer_guides/type-safe-api/index.md

```diff
@@ -1,14 +1,19 @@
 # Type Safe API
 
+_question: What does this refer to?_
 Define your APIs using [Smithy](https://smithy.io/2.0/) or [OpenAPI v3](https://swagger.io/specification/), and leverage the power of generated client and server types, infrastructure, documentation, and automatic input validation!
 
-This package vends a projen project type which allows you to define an API using either [Smithy](https://smithy.io/2.0/) or [OpenAPI v3](https://swagger.io/specification/), and a construct which manages deploying this API in API Gateway, given an integration (eg a lambda) for every operation.
+The _type-safe-api_ package provides a projen project type which allows you to define an API using either [Smithy](https://smithy.io/2.0/) or [OpenAPI v3](https://swagger.io/specification/), and a construct which manages deploying this API in an API Gateway, provided an integration (eg a lambda) exists for every operation.
```

Comment:
> Perhaps we get rid of the ", provided an integration (eg a lambda) exists for every operation" here? Integrations are explained later so it might just add confusion :)

removed